proto/storage/cluster.proto

```diff
@@ -113,11 +113,18 @@ message StaticClusterConfig {
   bool admission_controller_fail_on_error = 11;
 }
 
+message AutoLockProcessBaselines {
+  // More fields can be added later to control the feature at the
+  // namespace level
+  bool enabled = 1;
+}
+
 // The difference between Static and Dynamic cluster config is that Dynamic values are sent over the Central to Sensor gRPC connection. This has the benefit of allowing for "hot reloading" of values without restarting Secured cluster components.
 message DynamicClusterConfig {
   AdmissionControllerConfig admission_controller_config = 1;
   string registry_override = 2;
   bool disable_audit_logs = 3;
+  AutoLockProcessBaselines auto_lock_process_baselines = 4;
```

Comment:
autolock_config should be sufficient IMO, or if you prefer autolock_process_baselines_config

Reply:
`autolock_config` might be too broad. Auto-locking might be applied to networking baselines in the future.